demos/object_detection_demo_faster_rcnn/main.cpp

```diff
@@ -373,8 +352,7 @@ int main(int argc, char *argv[]) {
 
         const float* detection = static_cast<PrecisionTrait<Precision::FP32>::value_type*>(output_blob->buffer());
 
-        std::vector<std::vector<int> > boxes(batchSize);
-        std::vector<std::vector<int> > classes(batchSize);
+        std::vector<std::vector<DetectedObject>> objects(batchSize);
```

Comment:
With this line `DetectedObject` gets an excuse why it is not still deleted form common.hpp :)

But I doubt we need `DetectedObject` here. A rectangle can be drawn just after extracting its coordinates in the same loop. This approach simplifies the demo.

Reply:
What's your problem with `DetectedObject`? :-) I think that's actually a pretty useful thing to have, since this structure reoccurs in many models.

In this case you're right, though. I simplified the demo as you suggested.

Reply:
> I think that's actually a pretty useful thing to have, since this structure reoccurs in many models.

Yes, but each time you need a slightly different struct in each particular case. Demos that need that kind of struct usually redesign it. For example, `difficult` field in `DetectedObject` is not usually needed.